.github/workflows/unit-tests.yml

```diff
@@ -36,3 +36,106 @@ jobs:
     - name: Run Unit Tests
       run: |
         make check
+  MacOS:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install packages
+        run: |
+          brew install automake cppunit libtool
+
+      - name: Compile and Test
+        run: |
+          autoreconf -fiv
+          ./configure
+          make
+          make check
```

Comment:
We should not have different steps from the original.

Reply:
Done